lib/irb/command/edit.rb

```diff
@@ -8,6 +8,8 @@ module IRB
 
   module Command
     class Edit < Base
+      include IRB::Command::RubyArgsExtractor
```

Comment:
I think this one was missed?

```suggestion
      include RubyArgsExtractor
```